.github/workflows/release-management.yml

```diff
@@ -0,0 +1,132 @@
+name: Release Management
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Install
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+
+      - name: Test
+        run: npm test
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: docs
+          path: dist/docs
+
+  update_release_draft:
```

Comment:
I couldn't get this job working yet. It says the config is invalid but I'm 99% sure it's there and its's valid. I couldn't find any help nor in the docs (@mbohal), nor with Google. I believe it might start working once the 'release-drafter.yml' config is merged into `master`.